src/command/cmd_funcs.c

```diff
@@ -9431,10 +9459,31 @@ cmd_slashguard(ProfWin* window, const char* const command, gchar** args)
     return TRUE;
 }
 
+gchar*
+_prepare_filename(gchar* url, gchar* path)
+{
+    // Ensure that the downloads directory exists for saving cleartexts.
```

Comment:
I think we use `// comment` quite often.

Interestingly enough I often use what you described under 'background' exactly in the opposite way :D Because with `/* */` it's easier to outcomment whole blocks.

Reply:
I guess it's resolved then :)